src/service.rs

```diff
@@ -646,10 +640,11 @@ where
         &mut self,
         request: ReceivingKeyRequest,
     ) -> Result<Vec<String>, ()> {
-        let response = self.state.lock().signer[request.network].receiving_keys(request.message);
-        let keys = response
+        let response = self.state.lock().signer[request.network].address();
+        let responses = vec![response];
```

Comment:
same for this return type

Reply:
Requests now are just GET as we don't have multiple addresses per account anymore

Reply:
I think ui is using `get_receiving_keys`, If we return `String`, then ui also need to change.

Reply:
Hm should probably discuss with Alex when its connected to the new UI